tests/Doctrine/Tests/ORM/Functional/Ticket/GH8607Test.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\ManyToOne;
+use Doctrine\ORM\Mapping\OneToMany;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use Exception;
+
+class GH8607Test extends OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        try {
+            $this->_schemaTool->createSchema([
+                $this->_em->getClassMetadata(GH8607ParentEntity::class),
+                $this->_em->getClassMetadata(GH8607ChildEntity::class),
+            ]);
+        } catch (Exception $e) {
+            // skip errors
+        }
```

Comment:
Please apply a change similar to what I did in https://github.com/doctrine/orm/pull/8962 :pray:

Reply:
done